### PR TITLE
Fix for openssl issue during build.

### PR DESCRIPTION
**Pull request by roryk**
* [X] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

This fixes the local building for me, it wasn't picking up openssl during the build. I could be a moron though, so a check would be a good plan before merging.

### CODE REVIEW

**Comment by bgruening** [comment]
It makes a lot sense to me. I just don't understand why this worked before. I will leave this open so other can chime in.

**Comment by roryk** [comment]
Me too for both statements. :)

File: recipes/ucsc-gff3togenepred/include.patch [hunk 1]
```diff
@@ -5,7 +5,7 @@
  
  HG_DEFS=-D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_GNU_SOURCE -DMACHTYPE_${MACHTYPE}
 -HG_INC+=-I../inc -I../../inc -I../../../inc -I../../../../inc -I../../../../../inc
-+HG_INC+=-I../inc -I../../inc -I../../../inc -I../../../../inc -I../../../../../inc -I /anaconda/include
++HG_INC+=-I../inc -I../../inc -I../../../inc -I../../../../inc -I../../../../../inc -I /anaconda/include -I ${PREFIX}/include
```
Comment (johanneskoester) [inline recipes/ucsc-gff3togenepred/include.patch]: Do we really need the /anaconda/include? I would prefer the recipe to be independent of our setup. I think $PREFIX/include should be sufficient.

Comment (roryk) [inline recipes/ucsc-gff3togenepred/include.patch]: Sweet, deleting unnecessary stuff sounds good to me.

**Comment by johanneskoester** [comment]
Looks good, thanks!

**Pull request merged**